### PR TITLE
DM-24534: Add hsc source table schema

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: yml/hsc.yaml [hunk 1]
```diff
@@ -3205,3 +3205,770 @@ tables:
...
+    mysql:datatype: FLOAT
+    fits:tunit: pixel
+    ivoa:ucd: stat.error:pos.cartesian.y
+  - name: ra
+    "@id": "#Source.ra"
+    datatype: double
+    description: Calibrated RA-coordinate of the center of the source transformed
+      from xy.
+    mysql:datatype: DOUBLE
+    fits:tunit: arcsec
```
Comment (yalsayyad) [inline yml/hsc.yaml:3305]: degrees

File: yml/hsc.yaml [hunk 2]
```diff
@@ -3205,3 +3205,770 @@ tables:
...
+    ivoa:ucd: pos.eq.ra
+  - name: decl
+    "@id": "#Source.decl"
+    datatype: double
+    description: Calibated Decl-coordinate of the center of the source transformed
+      from xy.
+    mysql:datatype: DOUBLE
+    fits:tunit: arcsec
```
Comment (yalsayyad) [inline yml/hsc.yaml:3313]: degrees

File: yml/hsc.yaml [hunk 3]
```diff
@@ -3205,3 +3205,770 @@ tables:
...
+    ivoa:ucd: pos.eq.dec
+  - name: sky
+    "@id": "#Source.sky"
+    datatype: double
+    description: Estimated sky background at the position (centroid) of the source.
+    mysql:datatype: DOUBLE
+    fits:tunit: nmgy/asec^2
```
Comment (yalsayyad) [inline yml/hsc.yaml:3320]: Maybe just delete unites for the sky values.  It's def not `nmgy/asec^2`

File: yml/hsc.yaml [hunk 4]
```diff
@@ -3205,3 +3205,770 @@ tables:
...
+    description: Calibrated flux for Point Source model.
+    mysql:datatype: DOUBLE
+    fits:tunit: nmgy
+    ivoa:ucd: phot.count
+  - name: PsFluxErr
+    "@id": "#Source.PsFluxErr"
+    datatype: double
+    description: Uncertainty of psFlux.
+    mysql:datatype: DOUBLE
+    fits:tunit: nmgy
```
Comment (yalsayyad) [inline yml/hsc.yaml:3339]: nanojansky

File: yml/hsc.yaml [hunk 5]
```diff
@@ -3205,3 +3205,770 @@ tables:
...
+    datatype: double
+    description: Uncertainty of sky.
+    mysql:datatype: DOUBLE
+    fits:tunit: nmgy/asec^2
+  - name: PsFlux
+    "@id": "#Source.PsFlux"
+    datatype: double
+    description: Calibrated flux for Point Source model.
+    mysql:datatype: DOUBLE
+    fits:tunit: nmgy
```
Comment (yalsayyad) [inline yml/hsc.yaml:3332]: nanojansky

File: yml/hsc.yaml [hunk 6]
```diff
@@ -3205,3 +3205,770 @@ tables:
...
+  - name: ApFlux_3_0_instFlux
+    "@id": "#Source.ApFlux_3_0_instFlux"
+    datatype: double
+    description:
+    mysql:datatype: DOUBLE
+  - name: ApFlux_3_0_instFluxErr
+    "@id": "#Source.ApFlux_3_0_instFluxErr"
+    datatype: double
+    description:
+    mysql:datatype: DOUBLE
```
Comment (yalsayyad) [inline yml/hsc.yaml:3554]: If we need to save space, all these ApFlux's can tolerate being single precision REALs

Comment (hsinfang) [reply yml/hsc.yaml]: I am changing all Source.ApFlux* to FLOAT then.

**Pull request merged**